uwblueprint/community-fridge-kw
# fix: update monthly donation logic + getting next recurring donation id logic

## Brief description. What is this change? 

### [Monthly Donation: Set Every 4 Weeks](https://www.notion.so/uwblueprintexecs/Monthly-Donation-Set-Every-4-Weeks-2b550807c9a343369c389c885871d93e)



## Implementation description. How did you make this change?
* Changed backend monthly donation logic to schedule donations 4 weeks/28 days 
* Changed drop down label in scheduling donation flow from "Monthly" to "Monthly (every 4 weeks)"
* Updated logic for getting next `recurringDonationId` value



## Steps to test
1. Create a recurring monthly donation through the UI
2. Use Postman to send a `GET` request to `\scheduling` endpoint and check that the donation start times are 4 weeks apart
3. Create more recurring donations
4. Check that each set of recurring donations has a unique `recurringDonationId` value




## Checklist
